shared/scss/popup.scss

```diff
@@ -748,6 +728,9 @@ body {
     }
 }
 
+@import "views/hero";
```

Comment:
Since this isn't a standalone view, maybe it doesn't make sense to have it in a `views` directory. Any names you prefer? We use `objects` in the core repo, but that's somewhat nebulous. `subviews` is used elsewhere to mean something else. `components`?

Reply:
Also, maybe we want to abstract away the template for the hero section into its own thing, but that's not trivial because of the `isCalculating` logic. Maybe we can do it in a separate PR (or file it away for a later date).